ui/job-view/JobView.jsx

```diff
@@ -162,6 +165,14 @@ class JobView extends React.Component {
     document.getElementById('favicon').href = thFavicons[status] || thFavicons.open;
   }
 
+  // If showing is a boolean, then set to that value.  If it's not, then toggle
+  showOnScreenShortcuts(showing) {
+    const { showShortCuts } = this.state;
+    const newValue = typeof showing !== 'boolean' ? !showShortCuts : showing;
```

Comment:
Would this be marginally easier to read without the negation? (ie `===` and the expressions swapped)

Reply:
good call, yeah.  :)